lib/WebService/MinFraud/Record/Device.pm

```diff
@@ -74,6 +86,14 @@ This is the date and time of the last sighting of the device on the specified
 IP address for your user account. The string is formatted in the ISO 8601
 combined date and time in UTC.
 
+=head2 session_age
+
+A session length (in seconds) consisting of a floating point number.
+
+=head2 session_id
+
+A session id consisting of a string up to 255 characters in length.
```

Comment:
Again, I might make this a bit more specific, e.g., mention that it should be an ID that uniquely identifies the visitor's session on their site.